dbt/adapters/athena/impl.py

```diff
@@ -56,59 +56,72 @@ def convert_number_type(cls, agate_table: agate.Table, col_idx: int) -> str:
     def convert_datetime_type(cls, agate_table: agate.Table, col_idx: int) -> str:
         return "timestamp"
 
+    @classmethod
+    def parse_lf_client_response(
```

Comment:
Could you please unit test this? I think that is pretty easy as you can provide a mock response from lakeformation. Ideally when moto provide mocking capabilities for LF should be even easier.

Reply:
sure, I've added unit tests, pls review